html/changelogs/archive/2022-08.yml

```diff
@@ -37,3 +37,17 @@
   - bugfix: Fixed a bug where if you join too early to the server your fps was 20
       instead of 60 (or other number you've set).
   - bugfix: Fixed a bug that prevented all removed organs from rotting.
+2022-08-17:
+  nianoru:
+  - bugfix: All security tier RIGs now have a manuervable jetpack.
+  - balance: Leaper's leap ability speed is now slightly increased.
+  - bugfix: 'USG Ishimura: EVA ADS is now fully indestructable.'
+  - bugfix: 'The Colony: all bridge shutter controls are fixed.'
+2022-08-18:
+  nianoru:
+  - rscadd: 'The Colony: crew monitoring console has been added to bridge.'
+  - rscadd: 'The Colony: a bucket has been added to botany.'
+  - rscadd: 'The Colony: medbay now has flashdrives for surgery.'
+  - rscadd: 'The Colony: tcomms circuit boards added to engineer fabricator room.'
+  - balance: 'The Colony: research and development now starts with two fifty-stack
+      steel total.'
```

Comment:
I told you not to compile the changelog this time. You are not the only contributor and if anyone else compiles it as well it will cause a merge conflict

Reply:
Will change that in the next commit, sorry.